packages/edit-site/src/components/header-edit-mode/style.scss

```diff
@@ -16,7 +16,12 @@ $header-toolbar-min-width: 335px;
 		display: flex;
 		border: none;
 		align-items: center;
+		flex-grow: 1;
 		flex-shrink: 2;
+
+		// Account for the site hub, with is 60x60px.
+		flex-basis: calc(37.5% - 60px);
```

Comment:
What is 37.5%?

Reply:
25% (the center) with 37.5% for left and right (left is technically 37.5 - 60px, the width of the site icon). That way it's actually centered, while allowing room for additional icons on the right.